stubs/webpack.mix.js

```diff
@@ -15,4 +16,11 @@ mix.js('resources/js/app.js', 'public/js')
     .postCss('resources/css/app.css', 'public/css', [
         require('postcss-import'),
         require('tailwindcss'),
-    ]);
+    ])
+    .webpackConfig({
+        resolve: {
+            alias: {
+                '@': path.resolve(__dirname, 'resources/js/')
```

Comment:
I wonder if this should be shortened to:

```js
'@': path.resolve('resources/js'),
```

(That's what I've [always done](https://github.com/inertiajs/pingcrm/blob/master/webpack.mix.js#L40).)

Reply:
Just tested, it works 👍
Pushed the update.